README.md

```diff
@@ -3,6 +3,16 @@
 This repository contains the code that is referred to in the tutorial 
 [Integrating Rasa with knowledge bases](https://blog.rasa.com/integrating-rasa-with-knowledge-bases/).
 
+## Outline
+
+1. Requirements
```

Comment:
```suggestion
1. [Requirements](#Requirements)
```

Reply:
think  you can link to sections this  way https://gist.github.com/asabaylus/3071099